src/components/DateInput/YearAndMonthSelector.jsx

```diff
@@ -54,11 +54,11 @@ YearAndMonthSelector.propTypes = {
   /** `react-day-picker` localeUtils */
   localeUtils: PropTypes.object.isRequired,
 
-  /** Year range start as JS `Date` object */
-  startYear: PropTypes.instanceOf(Date).isRequired,
+  /** Year range start as number */
```

Comment:
was this wrong, or did the api change?

Reply:
I think API was changed during development of the same PR that introduced it:
https://github.com/cbinsights/form-design-system/pull/420/files#diff-dce376a1ed3f7822a60ccf0e5358aa694ef800993d556fa7d2e7ad9f7028cd21R21
In any case, you can see that `getYearRange` linked above always returns those years as numbers.